app/src/main/java/com/ciandt/dragonfly/example/features/feedback/FeedbackActivity.kt

```diff
@@ -285,7 +285,6 @@ class FeedbackActivity : BaseActivity(), FeedbackContract.View {
         positiveButton.visibility = View.GONE
         negativeButton.visibility = View.GONE
         underRevision.visibility = View.VISIBLE
-        footer.visibility = View.VISIBLE
```

Comment:
I thought the problem here was different. I thought it should only not be visible in case there aren't other objects classified, no? Will this code only keep the same state as the component was before (if visible, keeps visible; if invisible, keeps invisible)?